bin/rake

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env ruby
+require_relative '../config/boot'
+require 'rake'
+Rake.application.run
```

Comment:
Why is a wrapper like this needed? Perhaps it's not passing down RAILS_ENV?

Reply:
The content was auto generated  with the file when I ran the update task. Will look into that. Thanks.

Reply:
Ah cool, also see how we do it in plots2 -- https://github.com/publiclab/plots2/pull/1799 - and maybe @Souravirus can take a look from time to time?

Reply:
This is an overview of all updates! https://github.com/publiclab/mapknitter/issues/305#issuecomment-467569596

Reply:
Thanks Jeff